### PR TITLE
Use an env variable instead of flag to switch on vnext.

### PR DESCRIPTION
**Pull request by aboodman**
I'm not usually a fan of environment variables but there are just too many different programs involved, the flag is a major pita.

### CODE REVIEW

File: cmd/noms/noms.go [hunk 1]
```diff
@@ -68,12 +67,12 @@ func main() {
 	for _, cmd := range commands {
 		if cmd.Name() == args[0] {
 			flags := cmd.Flags()
-			version.RegisterVersionFlags(*flags)
```
Comment (aboodman) [inline cmd/noms/noms.go]: The command line utilities and programs were reverted back to how they were before my prior change.

**Comment by aboodman** [comment]
cc @rafael-atticlabs

**Comment by rafael-atticlabs** [comment]
heh. LGTM. Just submitted this: https://github.com/attic-labs/noms/issues/3040.

**Comment by aboodman** [comment]
Gonna do this a different way.

**Pull request closed without merging**